### PR TITLE
feat(inlay): show virtual variable text

### PR DESCRIPTION
**Pull request by iamxiaojianzheng**
about this https://github.com/mistweaverco/kulala.nvim/issues/209

### CODE REVIEW

**Comment by gorillamoe** [comment]
This assumes people using TS. I would like to have this optional. I can look into some refactoring for that.

File: lua/kulala/inlay/init.lua [hunk 1]
```diff
@@ -44,4 +51,66 @@ M.show = function(t, linenr)
...
+
+M.toggle_virtual_variable = function()
+  M.show_virtual_variable_text = not M.show_virtual_variable_text
+
+  if M.show_virtual_variable_text then
+    add_virtual_variable_text()
+    vim.api.nvim_create_augroup(VV_GROUP_NAME, { clear = true })
+    vim.api.nvim_create_autocmd({ "BufEnter", "TextChanged", "TextChangedI" }, {
+      group = VV_GROUP_NAME,
+      pattern = "*.http",
```
Comment (gorillamoe) [inline lua/kulala/inlay/init.lua:108]: I think 🤔 we might also include *.rest here

**Review by gorillamoe — CHANGES_REQUESTED** [review]

**Comment by iamxiaojianzheng** [comment]
> This assumes people using TS. I would like to have this optional. I can look into some refactoring for that.

Look forward to your optimization

File: lua/kulala/inlay/init.lua (hunk 1 above)
Comment (iamxiaojianzheng) [reply lua/kulala/inlay/init.lua]: has done

**Pull request closed without merging**